crypto_sign/falcon-1024/META.yml

```diff
@@ -22,9 +22,9 @@ auxiliary-submitters:
   - Zhenfei Zhang
 implementations:
   - name: clean
-    version: 20230207
+    version: 20211101 with PQClean patches
```

Comment:
Do we know if the Falcon team has any other name for this? E.g. version 2.1 or something like that?

Reply:
This is the version number of the Falcon zip file from `falcon-sign.info`, which my source package has no differences with other than our own patches.

Reply:
Oh, so there are no other changes to Falcon (algorithmically) since then?  Okay then.

Reply:
Yeah I think the spec speaks of version 1.2?